multicluster/controllers/multicluster/commonarea/remote_common_area.go

```diff
@@ -189,6 +162,36 @@ func NewRemoteCommonArea(clusterID common.ClusterID, clusterSetID common.Cluster
 	return remote, nil
 }
 
+func GetRemoteConfigAndClient(secretObj *v1.Secret, url string, clusterID common.ClusterID, clusterSet *multiclusterv1alpha1.ClusterSet, scheme *runtime.Scheme) (*rest.Config,
+	manager.Manager, client.Client, error) {
+	crtData, token, err := GetSecretCACrtAndToken(secretObj)
```

Comment:
It's not introduced by this PR, but this `GetSecretCACrtAndToken` might be renamed to `getSecretCACrtAndToken` if it's not used by external package, could you double check? thanks.